tchannel-core/src/main/java/com/uber/tchannel/handlers/RequestRouter.java

```diff
@@ -119,6 +119,10 @@ protected void messageReceived(final ChannelHandlerContext ctx, final Request re
         // Get handler for this method
         RequestHandler handler = this.getRequestHandler(service, endpoint);
         if (handler == null) {
+            handler = topChannel.getDefaultHandler();
```

Comment:
You can treat this default handler as a normal user handler. It takes request and returns response. It still returns back to the requestRouter.